test/malli/swagger_test.cljc

```diff
@@ -276,3 +276,142 @@
                                                        [:zip int?]
                                                        [:country "Country"]]]]]]}}
            "Order"]))))
+
+(deftest swagger-spec-test
+  (testing "generates swagger for ::parameters w/ basic schema"
+    (let [registry (merge (m/type-schemas)
+                          {::body [:string {:min 1}]})]
+      (is (= {:definitions {::body {:minLength 1, :type "string"}},
+              :parameters [{:description "",
+                            :in "body",
+                            :name "body",
+                            :required true,
+                            :schema {:$ref "#/definitions/malli.swagger-test~1body",
+                                     :definitions {::body {:minLength 1, :type "string"}}}}]}
```

Comment:
should it be `:definitions {"malli.swagger-test~1body" ...}`?

Reply:
That's a good question. I think it doesn't need to be because by the time it gets serialized to JSON it gets converted to that. The one aspect I'm a little surprised by is that the `/` -> `~1` conversion is happening. But maybe that's just because it's being converted by JSON-Schema-aware code? Probably worth a little more investigation.

Reply:
Jsonista and other clojure json libraries just convert `:foo.bar/baz` to `"foo.bar/baz"`. Grepping for `~` doesn't give me any hits from the malli or reitit repositories.

Reply:
Oh! I remember now. We don't want those to have `/` changed to `~1` because they aren't pointers, they're just regular old JSON object keys. You have to encode `/` as `~1` _only_ in JSON pointers (a.k.a. the values of `$ref` keys). [See here](https://opis.io/json-schema/2.x/pointers.html).

So yeah, the qualified keywords get expanded into the appropriate string when serialized to JSON.

Reply:
Oh right. That makes sense. Could you add that link as a comment to  `-ref`? I think this is ready merge after that!

Reply:
Actually, I'll just do that for you so we can avoid the round-trip.